course6/toyapp_mikhail/displaylib/surface.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+
+#include "sdlobject.h"
+
+#include <SDL2/SDL.h>
+
+#include <cstdint>
+
+namespace Display {
+
+	class Vector2D
+	{
+	public:
+		double x, y;
+	};
+
+	class Vector3D
+	{
+	public:
+		double x, y, z;
+	};
+
+	class Matrix
```

Comment:
Put the classes in separate files please

Reply:
Sure! Will be done